packages/integration-karma/test/mixed-shadow-mode/transitivity/index.spec.js

```diff
@@ -0,0 +1,137 @@
+import { createElement, setFeatureFlagForTest } from 'lwc';
+
+import DefaultExtendsNative from 'x/defaultExtendsNative';
+import LightContainer from 'x/lightContainer';
+import NativeContainer from 'x/nativeContainer';
+
+function isSyntheticShadowRootInstance(sr) {
+    return Boolean(sr && /function SyntheticShadowRoot/.test(sr.constructor.toString()));
+}
+
+function isNativeShadowRootInstance(sr) {
+    return Boolean(sr && !isSyntheticShadowRootInstance(sr));
+}
+
+if (!process.env.DISABLE_SYNTHETIC) {
```

Comment:
With the recent change, the double negation here makes it harder to read. The condition above reads: "With synthetic shadow loaded", correct?

Reply:
Yeah, this is unfortunate. Using the polyfill is still the default use case so the name becomes a negative when we exclude it.

Reply:
Assigned the double negation to a variable with a descriptive name.